src/main/sensors/gyro.c

```diff
@@ -181,8 +180,12 @@ PG_RESET_TEMPLATE(gyroConfig_t, gyroConfig,
     .gyroMovementCalibrationThreshold = 48,
     .gyro_sync_denom = GYRO_SYNC_DENOM_DEFAULT,
     .gyro_lpf = GYRO_LPF_256HZ,
-    .gyro_soft_lpf_type = FILTER_PT1,
-    .gyro_soft_lpf_hz = 90,
+    .gyro_lowpass_type = FILTER_PT1,
```

Comment:
The non-appending changes to this parameter group require incrementing of the parameter group version at the end of  ` PG_REGISTER_WITH_RESET_TEMPLATE()` above.

Reply:
In de1baed50da67d45d214a005fc256a485438fc59